tutorials/W2D5_ClimateResponse-AdaptationImpact/W2D5_Tutorial1.ipynb

```diff
@@ -2,1238 +2,791 @@
    ]
   },
```

Comment:
Whats going on with the above image file?

Reply:
Which image filer?

I can see one this only
![image](https://github.com/neuromatch/climate-course-content/assets/61898308/f5c5b2b3-21c3-470f-9592-233650bdd025)